### ISSUE TITLE
Bulk Upload

### ISSUE THREAD

**Issue by holysanta** [kartoza/parcel_plugin#59]
How can a user bulk upload coordinates for a parcel?

**Comment by NyakudyaA** [comment]
@holysanta can you be more specific. If a user has access to the database then SQL would be the best option

**Comment by holysanta** [comment]
....fortunately, not all users have db access! Data entry officers are issued the computation sheets covering large perimeters. Bulk upload from the user interface could help...

**Comment by NyakudyaA** [comment]
@holysanta I think there are tools already in QGIS to do this. One example of such a tool is a plugin called [Feature Loader](https://cbsuygulama.wordpress.com/2016/04/06/feature-loader-usage/). This facilitates bulk inserts into any loaded tables. 

For instance, if a user is using the plugin and you have a couple of beacons you need to add. 
* You first create a CSV table with the same fields as used by the beacon table eg beacons_samples.csv
Example below:
![sample](https://user-images.githubusercontent.com/2510900/36467326-e7d213e6-16e6-11e8-9290-843a06d2bd73.png)
*  Populate the CSV with your data.
* Load the CSV into QGIS as a layer.
* Then use the feature loader plugin.
* Select the source `Target To` as the beacon layer loaded by the plugin and select `Source Layer` as 
your loaded CSV table (beacons_samples)
* Then start the upload process.

**Comment by holysanta** [comment]
Yes @NyakudyaA  that is what is being used now. Would it be a bit more user friendly if this was a feature available in the plugin?
